app/grandchallenge/products/templates/products/partials/tooltip_CE.html

```diff
@@ -0,0 +1,2 @@
+{% load static %}
```

Comment:
I don't think this is needed. Nor is it in the other two tooltip templates.

Reply:
fixed